### DISCUSSION TITLE
IOTA Legacy Migration

### DISCUSSION THREAD

**Discussion by FedeCosmico** [iotaledger/firefly#8114] [Help & Support]
Is this migration mandatory? The reality is that I don't really understand how to do it. I follow the steps but then I don't know how to move forward. I'm not an expert on this. Thanks for your contributions.

**Comment by begonaalvarezd** [comment]
Attention

In case you still have funds on old IOTA Seeds (Trinity Wallet, IOTA Light Wallet) and have NOT migrated those Funds to Firefly Wallet yet, please find here the instructions on how you can reaccess those Funds: https://blog.iota.org/iota-legacy-migration-tool/

If all your IOTAs are already held via FIREFLY Wallet (or Bloom or Tanglepay), you don't need to do anything and can ignore this message.

**Comment by FedeCosmico** [comment]
Ok, that's if I have it solved and I have my firefly. My query was why we are talking about migration to IOTA legacy. Should I do something with that? El 29 feb 2024, a la(s) 11:13, Begoña Álvarez de la Cruz ***@***.***> escribió:﻿
Attention
In case you still have funds on old IOTA Seeds (Trinity Wallet, IOTA Light Wallet) and have NOT migrated those Funds to Firefly Wallet yet, please find here the instructions on how you can reaccess those Funds: https://blog.iota.org/iota-legacy-migration-tool/
If all your IOTAs are already held via FIREFLY Wallet (or Bloom or Tanglepay), you don't need to do anything and can ignore this message.

**Reply by begonaalvarezd** [reply]
The migration is not TO legacy but FROM legacy, if you have unmigrated funds in the old legacy network (before april 2021 or so) then you should migrate your funds

**Comment by FedeCosmico** [comment]
Now I understand perfectly, thank you very much for your time and information!

Greetings!
